src/mantine-form/src/use-form.ts

```diff
@@ -45,7 +46,7 @@ export function useForm<
   validate: rules,
 }: UseFormInput<Values, TransformValues> = {}): UseFormReturnType<Values, TransformValues> {
   const [touched, setTouched] = useState(initialTouched);
-  const [dirty, setDirty] = useState(initialDirty);
+  const [manualDirtyOverride, setDirtyManualOverride] = useState(initialDirty);
```

Comment:
Is there a reason to rename these variables? I think dirty and setDirty are less noisy

Reply:
The goal was to make it clearer that those are overrides of the normal `initial not equal current` behaviour. I can revert if you feel keeping the original naming this makes sense 👍